### PR TITLE
Create zarcdev.json

### PR DESCRIPTION
**Pull request by ZarcDev21**

### CODE REVIEW

File: sub-logs/zarcdev.json
Comment (ZarcDev21) [reply sub-logs/zarcdev.json]: Done

**Review by WilliamDavidHarrison — APPROVED** [review]

**Comment by WilliamDavidHarrison** [comment]
Please wait for another helper/maintainer to approve and merge these changes. (It requires 2 people to merge a PR)

**Pull request merged**
